packages/e2e-tests/src/specs/editor/publishPanel/adminUser.js

```diff
@@ -79,7 +79,7 @@ describe('Publish panel in document tab', () => {
     expect(optionListBeforeSearch).toBeDefined();
     expect(optionListBeforeSearch.length).toBeGreaterThan(1);
 
-    await page.keyboard.type('auth');
+    await page.keyboard.type('auth', { delay: 10 }); // was seeing some typing / input issues
```

Comment:
When/where did this happen? I've never seen this test fail.

Reply:
removed.